var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -148,16 +151,19 @@ class PyTensorflow(Package, CudaPackage):
     depends_on('py-backports-weakref@1.0rc1', type=('build', 'run'), when='@1.2.0:1.2.1')
```

Comment:
You're missing a dep on `py-clang`, which you'll need to create a package for.

Reply:
I just added this. It is somewhat surprising that this dependency is for clang-5.